packages/mui-material/src/Accordion/AccordionDataAttributes.ts

```diff
@@ -0,0 +1,13 @@
+enum AccordionDataAttributes {
+  /**
+   * Indicates whether the panel is opened
```

Comment:
```suggestion
   * Indicates whether the panel is opened.
```

Reply:
Side note, I was thinking of making a quick PR to fail the CI anytime any of those descriptions doesn't end with a dot. I suspect it's a quick win.

Reply:
I am reverting these changes, they are just here to help the reviewer :)